index.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to https://jobs.picnic.app/en/tech</title>
+<meta http-equiv="refresh" content="0; URL=https://jobs.picnic.app/en/tech">
```

Comment:
Immediately refreshes the page and redirects to the new jobs site.

I basically followed this guide. https://dev.to/steveblue/setup-a-redirect-on-github-pages-1ok7

Reply:
This will affect SEO since it doesn't use a 301 redirect and instead needs to parse the html. Maybe this page has a good ranking and we would lose that. 

Then the alternative is to set a custom domain for the github pages which is just the new website. 
![image](https://github.com/user-attachments/assets/20afb0e7-56b4-4f05-a967-46d7bf79384b)

Reply:
Not too worried about SEO for this page, as the whole point is that we want to make it less visible 👍